apps/wallet-mobile/src/i18n/locales/en-US.json

```diff
@@ -484,6 +484,48 @@
   "components.txhistory.txnavigationbuttons.receiveButton": "Receive",
   "components.txhistory.txnavigationbuttons.sendButton": "Send",
   "components.uikit.offlinebanner.offline": "You are offline. Please check settings on your device.",
+
+  "components.walletinit.txnavigationbuttons.continueButton": "Continue",
```

Comment:
all strings, should be migrated to <feature>.<useCase>.<key>, please check example